DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -36,5 +37,86 @@ public void Get_course_content_should_return_course()
             };
             result.Should().BeEquivalentTo(expectedCourse);
         }
+
+        [Test]
+        public void Get_progress_id_should_return_progress_id()
+        {
+            // When
+            const int candidateId = 9;
+            const int customisationId = 259;
+            var result = courseContentService.GetProgressId(candidateId, customisationId);
+
+            // Then
+            var expectedProgressId = 10;
+            result.Should().Be(expectedProgressId);
+        }
+
+        [Test]
+        public void Update_login_count_should_not_increment_login_if_no_new_session()
+        {
+            // When
+            const int candidateId = 9;
+            const int customisationId = 259;
+            var progressId = courseContentService.GetProgressId(candidateId, customisationId);
+            var expectedLoginCount = CourseContentTestHelper.GetLoginCount(progressId);
+            courseContentService.UpdateLoginCountAndDuration(progressId);
+            var result = CourseContentTestHelper.GetLoginCount(progressId);
+
+            // Then
+            result.Should().Be(expectedLoginCount);
+        }
+
+        [Test]
+        public void Update_duration_should_not_increment_login_if_no_new_session()
+        {
+            // When
+            const int candidateId = 9;
+            const int customisationId = 259;
+            var progressId = courseContentService.GetProgressId(candidateId, customisationId);
+            var expectedDuration = CourseContentTestHelper.GetDuration(progressId);
+            courseContentService.UpdateLoginCountAndDuration(progressId);
+            var result = CourseContentTestHelper.GetDuration(progressId);
+
+            // Then
+            result.Should().Be(expectedDuration);
+        }
+
+        [Test]
+        public void Update_login_count_should_increment_login_if_new_session()
+        {
+            // When
+            const int candidateId = 9;
+            const int customisationId = 259;
+            const int duration = 5;
+            const int active = 0;

```

Comment:
`active` isn't actually relevant to the test so can probably just always be set to 0 in the helper method 🙂